resolver/SuggestPackage.cc

```diff
@@ -291,6 +291,16 @@ bool SuggestPackage::tryPackageCorrections(core::Context ctx, core::ErrorBuilder
         for (auto match : missingImports) {
             pkgCtx.addMissingImportSuggestions(e, match);
         }
+
+        // going from an UnresolvedConstantLit to the full
+        // (with-scope) string is actually kinda annoying, so we're
+        // using the `loc` to get the original chunk of the file that
+        // includes the whole constant with scope
+        if (auto fullConstant = ctx.locAt(unresolved.loc).source(ctx)) {
+            // TODO (document me)
```

Comment:
what would you have liked to document here?

Reply:
Oh, that's a legacy of a slightly older approach—originally I was going to have two different error messages depending on whether we had one autocorrect (i.e. a "correct" one) or not, and I wanted to explain why. I don't think I need an extra comment there now.